roles/install_ocp/meta/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+galaxy_info:
+  role_name: redhat_cop.tower_utilities.ocp_install
+  author: Chris Renwick
+  description: Role to install Ansible Tower on Openshift
+  company: Red Hat
+
+  license: license (MIT)
+
+  min_ansible_version: 2.7
+
+  platforms:
+    - name: EL
+      versions:
+        - 7
+
+  galaxy_tags:
+    - linux
+    - system
+    - ansible
+
+dependencies:
+  - role: redhat_cop.tower_utilities.pre_tasks
+    vars:
+      tower_ocp_releases_url: "{{ tower_releases_url }}"
```

Comment:
@crenwick93 youve done this the wrong way around

It should read 
`tower_releases_url: "{{ tower_ocp_releases_url }}"`

And similar for the other var here

Reply:
I've tested it with 
```
tower_releases_url: "{{ tower_ocp_releases_url }}"
tower_setup_file: "{{ tower_ocp_setup_file }}"
```
And it worked without issues.

Reply:
Thanks, just pushed that change :)